src/server/routes/exports.js

```diff
@@ -4,11 +4,17 @@ const _ = require('lodash')
 
```

Comment:
don't think the exports routes are in-use

Reply:
Well, the code exists still and it theoretically could be called, so it needs to obey multitenant rules. If you want to kill it after this PR merges, that's cool 👍

Reply:
Hmm, it does seem in use: the main "Download Treasury Report" button on the dashboard links to `/api/exports`